typescript/src/schema/proto.ts

```diff
@@ -159,6 +161,26 @@ function longRunningMetadataType(
   );
 }
 
+// Detect non-AIP-compliant LRO method for DIREGAPIC, where the response type is customized Operation and the method is not polling method.
+// (TODO: summerji) Update the detector when DIREGAPIC LRO annotation merged in googleapis-discovery.
+function isDiregapicLRO(
+  packageName: string,
+  method: MethodDescriptorProto,
+  isDiregapic?: boolean
+): boolean {
+  const operationOutputType = toFullyQualifiedName(packageName, 'Operation');
+  return isDiregapic &&
+    method.outputType &&
+    method.outputType === operationOutputType &&
+    method.options?.['.google.api.http'] &&
+    !(
+      method.options?.['.google.api.http'].get ||
+      (method.name === 'Wait' && method.options?.['.google.api.http'].post)
+    )
+    ? true
```

Comment:
nit: there is no need to write `booleanValue ? true : false`, just `booleanValue` is enough.  So you can just remove lines 180 and 181.

Reply:
I removed line 180 and 181, but update `isDiregapic` with the type `boolean | '' | null | undefined`